example/Cargo.toml

```diff
@@ -17,10 +17,10 @@ libc = "0.2.79"
 byteorder = "1.3.2"
 log = "0.4.6"
 simple-logging = "2.0.2"
-nix = "0.16.1"
+nix = "^0.20.2"
```

Comment:
The same reason as above, the '^' should be deleted.

Reply:
Fixed.